src/Data/Profunctor/Linear.hs

```diff
@@ -56,13 +102,28 @@ class (SymmetricMonoidal m u, Profunctor arr) => Strong m u arr where
   second arr = dimap swap swap (first arr)
   {-# INLINE second #-}
 
+-- | This typeclass asserts the constraint on the profunctor @arr@ that
+-- function-like things of type @a `arr` b@ with certain properties
+-- can be extended to work on traversible structures. These are the properties
+-- these function-like things need: they need @Strong (,) ()@ and
```

Comment:
These four lines just repeats what the first line of the instance definition. I think we should just remove them.